process/tests/test_integration.py

```diff
@@ -0,0 +1,16 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+import pytest
+
+from . import common
+
+pytestmark = pytest.mark.integration
+
+
+@pytest.mark.usefixtures("dd_environment")
+def test_check(aggregator, check):
+    check.check(common.instance)
```

Comment:
Let's make a deepcopy, in case we add more tests and need to manipulate the instance. An `instance` fixture doing that like many other tests would be fine.